task/log.html

```diff
@@ -132,17 +131,17 @@
 var tap_total_time = /^# (\d+ TESTS FAILED|TESTS PASSED) \[([0-9]+)s on .*\]$/m
 var test_header_start = "# ----------------------------------------------------------------------"
 
-var entry_template = $("#TestEntry").html();
+var entry_template = document.querySelector("#TestEntry").innerHTML;
```

Comment:
> Thanks! I tested this by putting a raw log into /tmp/, adding this log.html, and running `python3 -m http.server`. This looks like the old log, but what's missing now is the expandability of every test. Failures are expanded by default (so that we can see what's going on), but succeeding tests are hidden. It's not the most important thing in the world, but how much effort is it to fix that?

That sounds like something we should fix, as I will find that annoying.

Reply:
> Next step: make all these `const`. I.e. one round of eslint --fix would probably work wonders grin

Yes, not sure if `eslint` can handle html files.

Reply:
Ok, I've now added a commit to cleanup var's.

Reply:
I think it makes sense to first merge this while keeping jQuery around and then porting it to bootstrap 5 which does not require jQuery.